docs/blockchain/blockchain.mdx

```diff
@@ -43,17 +43,16 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
     </div>
     <div class="col col--6">
       <p>
-        The Helium Blockchain is based on a new, novel work algorithm called Proof of Coverage
-        (PoC), and rewards miners in $HNT, the native token of the Helium blockchain. Officially
-        launched on July 29, 2019, the Helium Blockchain powers{' '}
+        The Helium blockchain is based on a new, novel work algorithm called Proof of Coverage
+        (PoC), and rewards miners in HNT, IOT, and MOBILE, the native tokens of the Helium
```

Comment:
I wouldn't say that it already rewards with IoT and Mobile. You can say something like soon will reward.